engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -38,7 +42,9 @@ class DeploymentPreparer(config: K8sDeploymentManagerConfig) extends LazyLogging
         (deploymentSpecLens composeLens GenLens[Deployment.Spec](_.replicas)).modify(modifyReplicasCount(determinedReplicasCount)) andThen
         (deploymentSpecLens composeLens GenLens[Deployment.Spec](_.template.metadata.name)).set(objectName) andThen
         (deploymentSpecLens composeLens GenLens[Deployment.Spec](_.template.metadata.labels)).modify(_ ++ labels) andThen
-        (templateSpecLens composeLens GenLens[Pod.Spec](_.volumes)).modify(_ ++ List(Volume("configmap", Volume.ConfigMapVolumeSource(configMapId)))) andThen
+        (templateSpecLens composeLens GenLens[Pod.Spec](_.volumes)).modify(_ ++ List(
+          Volume("configmap", Volume.ConfigMapVolumeSource(configMapId)),
+          Volume("shared-configmap", Volume.ConfigMapVolumeSource(SharedConfigmapName, optional = Some(true))))) andThen
```

Comment:
configMapId + SharedConfigmapName

Reply:
It has to be static name? Becase all containers will use it